app/views/users/show.html.erb

```diff
@@ -1,45 +1,32 @@
 <h2>Here details for a given user</h2>
 
-<h3>User 3</h3>
-<h4>Number of post: 4</h4>
+<section class='user'>
+  <figure >
+    <image class='user-image' src= '<%= @user.photo %>' alt="<%= @user.name %>">
+  </figure>
+  <div class='user-info'>
+    <h2>User: <%= @user.name %></h2>
+    <p>Number of post:<strong><%= @user.post_counter || 0 %></strong></p>
```

Comment:
Same here. The posts counter is not working. As you can see, there is one post but the Number of posts is still 0. Kindly fix this issue. ⬇️
![image](https://github.com/LarryIVC/larrys-blog/assets/33268927/7eb1096a-3c43-48a4-8aca-8a1a8bbbe574)

Reply:
Done